test/e2e/parallel/nstemplatetier_test.go

```diff
@@ -386,27 +392,38 @@ func TestFeatureToggles(t *testing.T) {
 				// Verify that the CRB is back
 				_, err = wait.For(t, memberAwait.Awaitility, &v1.ClusterRoleBinding{}).WithNameThat(crbName)
 				require.NoError(t, err)
+				// the noise CRB for unknown/disabled feature is still not created
+				err = wait.For(t, memberAwait.Awaitility, &v1.ClusterRoleBinding{}).WithNameDeleted(noiseCrbName)
+				require.NoError(t, err)
 			})
 		})
 	})
 }
 
-func withClusterRoleBinding(t *testing.T, otherTier *toolchainv1alpha1.NSTemplateTier, feature string) tiers.CustomNSTemplateTierModifier {
-	var tpl bytes.Buffer
-	err := template.Must(template.New("crb").Parse(viewCRB)).Execute(&tpl, map[string]interface{}{
-		"featureName": feature,
-	})
-	require.NoError(t, err)
+var unknownFeature = "unknown-feature"
```

Comment:
minor - should this be a const instead of var ?

Reply:
Sure: https://github.com/codeready-toolchain/toolchain-e2e/pull/1016/commits/79b82259425af05d3f1ddccb683f1eede5199bcb